### PR TITLE
DOC: Update README after BioViL-T

### PR DESCRIPTION
**Pull request by fepegar**

### CODE REVIEW

File: hi-ml-multimodal/README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ The notebook can also be run on [Binder][4], without the need to download any co
 The latest version can be installed using `pip`:
 
 ```console
-pip install "git+https://github.com/microsoft/hi-ml.git#subdirectory=hi-ml-multimodal"
+pip install --upgrade hi-ml-multimodal
```
Comment (ozan-oktay) [inline hi-ml-multimodal/README.md:20]: @fepegar This assumes that a new package is going to be released at the same time as we merge the PR in. I guess you are on it?

Comment (fepegar) [reply hi-ml-multimodal/README.md:20]: Sounds good. As soon as we merge, I'll publish. It'll take me a couple of minutes.

File: hi-ml-multimodal/README.md [hunk 2]
```diff
@@ -89,7 +114,7 @@ If you use our code or models in your research, please cite [our paper][7] (pres
 [4]: https://mybinder.org/v2/gh/microsoft/hi-ml/HEAD?labpath=hi-ml-multimodal%2Fnotebooks%2Fphrase_grounding.ipynb
 [5]: https://docs.conda.io/en/latest/miniconda.html
 [6]: https://aka.ms/biovil-models
-[7]: https://link.springer.com/chapter/10.1007/978-3-031-20059-5_1
-[8]: https://eccv2022.ecva.net/
+[7]: https://doi.org/10.1007/978-3-031-20059-5_1
+[8]: https://openreview.net/forum?id=5jScn5xsbo
```
Comment (ozan-oktay) [inline hi-ml-multimodal/README.md:118]: This link is private -- Please use the Arxiv link instead.

File: hi-ml-multimodal/README.md [hunk 3]
```diff
@@ -81,6 +82,30 @@ If you use our code or models in your research, please cite [our paper][7] (pres
     pages="1--21",
     isbn="978-3-031-20059-5"
 }
+
+@inproceedings{bannur2023learning,
+    title={Learning to Exploit Temporal Structure for Biomedical Vision{\textendash}Language Processing},
+    author={
+        Shruthi Bannur
```
[the remark this answers is not in the document]
Comment (fepegar) [reply hi-ml-multimodal/README.md]: Makes sense. Done.

File: hi-ml-multimodal/README.md (hunk 2 above)
Comment (fepegar) [reply hi-ml-multimodal/README.md]: Oops. Is it sensitive? Shall I revert and force-push?

**Pull request merged**